cogboard-app/src/test/kotlin/com/cognifide/cogboard/widget/type/logviewer/logparser/MockLogParserStrategyTest.kt

```diff
@@ -1,30 +1,22 @@
 package com.cognifide.cogboard.widget.type.logviewer.logparser
 
 import org.junit.jupiter.api.Test
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.TYPE
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.DATE
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.VARIABLE_DATA
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.HEADERS
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.TEMPLATE
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.PROVIDER
-import com.cognifide.cogboard.widget.type.logviewer.logparser.ParsedLog.Companion.MESSAGE
+import java.lang.AssertionError
 
-class MockLogParserStrategyTest {
+class MockLogParserTest {
```

Comment:
jeśli zmieniamy nazwę klasy, to zmieńmy też nazwę pliku, ale raczej sugeruję zostanie przy słowie Strategy

Reply:
Przywrócono.